docs/content/users/install/ddev-installation.md

```diff
@@ -266,6 +266,48 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
 
     It can be complicated to get private databases and files into Gitpod, so in addition to the launchers, the [`git` provider example](https://github.com/drud/ddev/blob/master/pkg/ddevapp/dotddev_assets/providers/git.yaml.example) demonstrates pulling a database and files without complex setup or permissions. This was created explicitly for Gitpod integration, because in Gitpod you typically already have access to private Git repositories, which are a fine place to put a starter database and files. Although [ddev-gitpod-launcher](https://drud.github.io/ddev-gitpod-launcher/) and the web extension provide the capability, you may want to integrate a Git provider—or one of the [other providers](https://github.com/drud/ddev/tree/master/pkg/ddevapp/dotddev_assets/providers)—for each project.
 
+=== "Codespaces"
+
+    ## GitHub Codespaces
+
+    DDEV is fully supported in [GitHub Codespaces](https://github.com/features/codespaces)
+
+    To run DDEV in codespaces you'll need to create a codespaces instance for your repo and add a `.devcontainer/devcontainer.json` to it as shown below. There are at least two ways to do this:
+
+    1. On your project repo on github.com click `Code → Codespaces → ... → Configure dev container` to create or edit a `devcontainer.json`. Edit or create the file to add the features shown in the example.
```

Comment:
Turn number into bullet since these are not sequential steps but alternate options. Re-style to follow documentation conventions and clarify expected UI path:

```suggestion
    * Visit your project’s GitHub repository and click the _Code_ dropdown → _Codespaces_ tab → _..._ to the right of “Codespaces” → _Configure dev container_. This will open a `devcontainer.json` file you can edit with the details below.
```